great_expectations/checkpoint/checkpoint.py

```diff
@@ -129,14 +129,14 @@ def get_substituted_config(
         if isinstance(config, dict):
             config = CheckpointConfig(**config)
 
-        # Necessary when using RuntimeDataConnector with SimpleCheckpoint
-        if isinstance(config.batch_request, BatchRequest):
-            config.batch_request = config.batch_request.get_json_dict()
-        runtime_kwargs_batch_request = runtime_kwargs.get("batch_request")
-        if isinstance(runtime_kwargs_batch_request, BatchRequest):
-            runtime_kwargs[
-                "batch_request"
-            ] = runtime_kwargs_batch_request.get_json_dict()
+        # # Necessary when using RuntimeDataConnector with SimpleCheckpoint
```

Comment:
@roblim Should we just delete this commented out code?..

Reply:
Addressed in 0b8aa845aaab17db9cdd5c3425ad8c445f7d313b